docs/cmd/tkn_bundle_pull.md

```diff
@@ -0,0 +1,51 @@
+## tkn bundle pull
+
+Push a new Tekton bundle
+
+### Usage
+
+```
+tkn bundle pull
+```
+
+### Synopsis
+
+Fetch a new Tekton Bundle from a registry and list the object(s) in the bundle. You can further narrow
+down the results by optionally specifying the kind, and then the name:
+
+	tkn bundle pull docker.io/myorg/mybundle:latest // fetches all objects
+	tkn bundle push docker.io/myorg/mybundle:1.0 tasks // fetches all Tekton tasks
```

Comment:
this doesn't work for me locally - it errors out with:

```
Error: second argument tasks is not a valid kind: ["Task" "Pipeline" "ClusterTask"]
```

Reply:
Poor usage message. My bad.